### PR TITLE
Method refactor dataloaders

### PR DESCRIPTION
**Pull request by nikkifalicov**
Reason: The previous logic to retrieve the dataloaders and apply image transformations needed to be modified individually for each dataset, which is clunky when we try to run labeled-set only training for multiple datasets.

Fix: Add in a get_dataloaders function to hyper_search.py that can create and return the dataloaders necessary for training without separate logic for each dataset. Created  different dataloaders for unlabeled data as well as CheXpert, to handle differences in label processing. Added in argument specifying path to unlabeled data.

Verification: Verified that data is processed and returned correctly by creating dataloaders for train, test, and validation (unlabeled data is not applicable for label-set only training at the moment, but logic to handle it was included), and manually saved outputs to qualitatively evaluate that the images were correct. Images are not included here since they contain sensitive medical information.

### CODE REVIEW

**Comment by CynthiaFeeney** [comment]
Added some TODO comments in the code. Code also needs to apply Flake8 formatting. Should be ready to merge after that.

File: src/hyper_search.py [hunk 1]
```diff
@@ -79,8 +81,138 @@ def get_dataloaders(args):
         tuple: 4 DataLoaders, which may be none
                train_loader, unlabel_loader, valid_loader, test_loader
     """
-    # TODO implement
-    return None
+    dataset_name = args.dataset_name
+    root_dataset_folder = args.root_dataset_path
+
+    train_csv_path = args.l_train_dataset_path
```
Comment (Lakshita110) [inline src/hyper_search.py]: We might want to use the variables we define here in the later dataloader code instead of referencing args.___ or take away these variables?

Comment (nikkifalicov) [reply src/hyper_search.py]: Good point. I think it might be better to just remove the variables, it would make calling the function a lot bulkier if we passed it in to the dataloader code.

**Comment by nikkifalicov** [comment]
Added in inheritance where it was possible and applied Flake8 formatting.
